pipeline/tests/json_stream_test.ml

```diff
@@ -48,4 +48,81 @@ let parse_two =
   Alcotest.(check (list parsed_location)) "jsons" expect parsed;
   ()
 
-let tests = [ parse_one; parse_two ]
+let parse_wrong =
+  Alcotest_lwt.test_case_sync "parse wrong" `Quick @@ fun () ->
+  let str = "{{{ this isn't json }}}" in
+  let parsed = Json_stream.json_full str in
+  let expect = [] in
+  Alcotest.(check (list parsed_location)) "jsons" expect parsed;
+  ()
+
+let parse_wrong_longer =
+  Alcotest_lwt.test_case_sync "parse_wrong_longer" `Quick @@ fun () ->
+  let str =
+    "{\n\
+    \      s =\n\
```

Comment:
I don't think ocaml supports `\` at the beginning of a line (?) (but it does work at the end to skip the line break)

Reply:
I didn't add those, ocamlformat did. Interestingly, it doesn't do that when I use the full-escape form of strings `{| foo |}`